### ISSUE TITLE
A symbol always presents in a centered input field in Edge browser

### ISSUE THREAD

**Issue by Roman-Sky** [noppa/text-security#3]
Hi. Thank you for this polyfill!
I faced a problem when tried to center a text in the input field and tested a result in Edge browser.
Check an example: https://jsfiddle.net/Frakir/L7uro1jn/9/
I've forked your demo and added these css rules:
```
input {
  text-align: center;
  width: 100%;
}
```
After that, as you can see, input field with empty value and placeholder contains a single symbol (disc or circle). There are no problems in Chrome.
I managed to fix it by adding an attribute to input: `placeholder=" "` (placeholder with a space) and setting a white color for text:
```
input::-webkit-input-placeholder {
   color: #fff;
}
```
That worked, but it looks like a sort of hack.

Tested on:
Edge 42.17134.1.0
EdgeHTML 17.17134

**Comment by noppa** [comment]
It seems that IE and Edge display a space character (U+0020) when the placeholder is empty. That's unnoticable in most fonts, but becomes visible in this font because it also has a glyph for the space character (passwords can contain spaces, after all).

I've opened an issue in their issue tracker, [here](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/18186001/).

Meanwhile, I can think of three options to go around this problem:

- Exclude the space character from the generated font  
   I don't think that's a good idea because passwords can contain spaces and they are also concealed in a normal password field.
- Use the solution you suggested.  
  I guess it's a bit of a hack, but hey, it works.
- Generate a new glyph for some character that isn't included in the font already, say, Ͱ (U+0370), and make that glyph be just a blank space. Then add `placeholder="Ͱ"` to the input field.  
   IMO that just sounds even more of a hack than your solution, so I don't think we should do that.

So, to summarize, we'll have to wait and see what the people working on IE/Edge say, there's not much we can do from within this library.

**Comment by noppa** [comment]
@Roman-Sky There's an update [in the issue I opened](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/18186001/).

Apparently, the issue will be fixed in an upcoming release of Edge.

**Comment by Roman-Sky** [comment]
@noppa Thank you for the research!
